### DISCUSSION TITLE
Resident Agents are not working on Servers

### DISCUSSION THREAD

**Discussion by thilinadias** [immense/Remotely#925] [Q&A]
Hi 
Good day to you!

I just installed Remotely on my Linux VM and used it for my internal servers. It's a very powerful tool and you guys have done an excellent job. I installed Resident Agents on the Server [it is a Server 2016] I couldn't install in on the server. can anyone help?

**Comment by CyberTechX** [comment]
How did you try to install it and what version of remotely are you using?
